spec/cjk/korean_spacing_spec.rb

```diff
@@ -122,7 +122,7 @@
     context "강물이 될때 까지 (space between  이 and 될,  때 and 까)" do
       it_behaves_like "good results for 강물이될때까지", '강물이 될때 까지'
     end
-    context "강물 이 될 때 까지 (as in the record)" do
+    context "강물 이 될 때 까지 (as in the record)", pending: 'includes newly cataloged catkey 12437806' do
```

Comment:
I sent an email to the Korean cataloger asking about the relevancy of ckey 12437806.